scripts/run-plugin-nuts.js

```diff
@@ -193,7 +197,7 @@ const qualifyPluginsWithNonUnitTests = (timeCreated, modules) => {
           ...qualifyPluginsWithNonUnitTests(timeCreated, pluginsToQualify),
         ];
       })
-      .filter((module) => module)
+      .filter((module) => module && module !== '')
```

Comment:
Just a small suggestion, don't have to implement it if it doesn't make sense but it would cover more falsy/truthy values.
```suggestion
      .filter((module) => Boolean(module))
```